K8s/api-deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
             - name: OS_AUTH_AUTHSTRATEGY
               value: keystone
             - name: OS_AUTH_URL
-              value: http://192.168.20.108/identity
+              value: http://KEYSTONE_IP/identity
```

Comment:
does YAML supports plain variable replacement?

Reply:
it needs something like `${ENV_VAR}`

Reply:
Yes.